rest/data/commit_queue.go

```diff
@@ -77,6 +77,19 @@ func (pc *DBCommitQueueConnector) FindCommitQueueByID(id string) (*restModel.API
 	return apiCommitQueue, nil
 }
 
+func (pc *DBCommitQueueConnector) CommitQueueRemoveItem(id, item string) (bool, error) {
+	cq, err := commitqueue.FindOneId(id)
+	if err != nil {
+		return false, errors.Wrapf(err, "can't get commit queue for id '%s'", id)
+	}
+	if cq == nil {
+		return false, nil
+	}
+
+	found, err := cq.Remove(item)
+	return found, errors.Wrapf(err, "can't remove item '%s'", item)
```

Comment:
I would argue for `false` here, for the same reason as above, unless returning the boolean is somehow important to the caller.

Reply:
the err here can be nil, presumably, so it's just awkward.

I'd just return cq.Remove() and not wrap this error.